packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```
@@ -11,49 +11,55 @@ governing permissions and limitations under the License.
 */
 
 @import '../commons/index.css';
-
-:root {
-  --spectrum-searchwithin-width: 250px;
+.spectrum-SearchWithin {
+  --spectrum-searchwithin-width: 240px;
+  --spectrum-searchwithin-searchfield-width: 128px;
```

we'll want to use the variable for this so it changes when it goes to large size

```suggestion
  --spectrum-searchwithin-width: var(--spectrum-global-dimension-size-3000);
  --spectrum-searchwithin-searchfield-width: var(--spectrum-global-dimension-size-1600);
```

---

should this be min-width?

---

Will do, thanks!!